util/src/main/java/tc/oc/pgm/util/bukkit/BukkitUtils.java

```diff
@@ -24,11 +24,25 @@ static Plugin getPlugin() {
   }
 
   Boolean isSportPaper = Bukkit.getServer().getVersion().contains("SportPaper");
+  Boolean isSpigot = Bukkit.getServer().getVersion().contains("Spigot");
+  Boolean isPaper = Bukkit.getServer().getVersion().contains("Paper");
```

Comment:
given that you may only be one of them at once, maybe just make an enum of supported server softwares, the enum can have the display name, the string to search for, and what nms hacks class to use (probs a method ref to the constructor, like `SportPaperNmsHacks::new`)

Reply:
That would certainly be cleaner than what I have now. I'll give it a try